rkt/run.go

```diff
@@ -313,3 +313,18 @@ func (e *envMap) Strings() []string {
 	}
 	return env
 }
+
+// privateNetList implements the flag.Value interface to allow specification
+// of -private-net with and without values
+type privateNetList string
```

Comment:
I would rather see this flag do the validation of the string format (and split into a list)